src/index.js

```diff
@@ -0,0 +1,146 @@
+const state = {
+    city: 'Honolulu',
+    lat: 21.3099,
+    long: 157.8581,
+    temp: 72,
+};
+
+const convertKtoF = (temp) => {
+    return (temp - 273.15) * (9 / 5) + 32;
+};
+
+const findLatitudeAndLongitude = () => {
+    let location = state.city;
+    axios.get('http://localhost:5000/location',
+    {
+        params: {
+            q: location,
+        },
+    })
+    .then( (response) => {
+        console.log(response);
+        state.lat = response.data[0].lat;
+        state.long = response.data[0].lon;
+        console.log(`response: ${state.city}`);
+        console.log('success in findLatitudeAndLongitude!', state.lat, state.long);
+        
+        getWeather();
+    })
+    .catch( (error) => {
+    console.log(`error: ${state.city}`);
+    console.log('error in findLatitudeAndLongitude!');
+    console.log(error);
+    });
+};
+
+const getWeather = () => {
+    axios
+      .get('http://localhost:5000/weather', {
```

Comment:
Same comment here about using a variable to reference the string literal as in `findLatitudeAndLongitude`